src/main/io/ledstrip.c

```diff
@@ -1057,6 +1044,17 @@ void pgReset_colors(hsvColor_t *instance)
     }
 }
 
+void pgReset_modeColors(modeColorIndexes_t *instance)
+{
+    memcpy(instance, &defaultModeColors, sizeof(defaultModeColors));
+}
+
+void pgReset_specialColors(specialColorIndexes_t *instance)
+{
+    memcpy(instance, &defaultSpecialColors, sizeof(defaultSpecialColors));
+}
+
```

Comment:
these will need updating now that the reset template code has been merged.  see other pg reset functions for details.